### PR TITLE
Feature/copy misc frontend

### PR DESCRIPTION
**Pull request by eurolife**
Filters, search and document cover page are integrated with the API (but still some things are unfinished.)

### CODE REVIEW

File: frontend/public/locales/en/searchResults.json [hunk 1]
```diff
@@ -2,17 +2,20 @@
...
+    "Region": "We use World Bank definitions; see https://www.worldbank.org/en/about/unit",
+    "Country": "The jurisdictions in which documents were published (for example India, or European Union). For more information, see our Methodology page",
+    "Sector": "The broad areas of economic activity to which the content of documents relates, for example, agriculture or transport. For more information, see our Methodology page",
+    "Instrument": "The interventions or measures contained in documents, for example, taxes or standards. For more information, see our Methodology page",
+    "Document type": "What types of document they are - for example, a law, strategy, or decree",
+    "Exact match": "By default, you'll be shown both exact matches and results which are related to your search. Tick this box to see only exact matches",
+    "Sort by": "Results can be sorted by relevance (default), by year of document publication, or alphabetically",
+    "Download PDF": "Click to download a CSV file of documents in the results list, providing their metadata and URLs to PDFs. Note - this will be available as soon as possible"
```
Comment (joel-wright) [inline frontend/public/locales/en/searchResults.json:19]: This doesn't look right

File: frontend/src/hooks/useDocumentDetail.ts [hunk 2]
```diff
@@ -3,18 +3,17 @@ import { fakePromise } from '../helpers';
...
 
   return useQuery(
     'document_detail',
-    () =>
-      // client.get(`/documents/${id}`)
-      fake(),
-    { refetchOnWindowFocus: false }
+    () => client.get(`/documents/${id}`, null),
+    // fake(),
+    { refetchOnWindowFocus: false, enabled: id !== undefined }
```
Comment (joel-wright) [inline frontend/src/hooks/useDocumentDetail.ts:17]: Is this kept around for future testing, or are we cleaning out the fake data in a future PR?

File: frontend/src/pages/account/index.tsx [hunk 3]
```diff
@@ -24,22 +25,22 @@ const Account = () => {
...
     id: null,
-    full_name: '',
+    names: '',
     email: '',
     is_active: false,
     is_superuser: false,
-    affiliation: '',
-    affiliation_type: '',
+    affiliation_organisation: '',
+    affiliation_type: [],
```
Comment (joel-wright) [inline frontend/src/pages/account/index.tsx:33]: the CSV you sent me also included "other affiliation"

File: frontend/src/pages/cookie-policy.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+                information and settings when you visit the website again,
+                resulting in a faster online experience for you.
+              </p>
+              <h3>Google Analytics</h3>
+              <p>__ga__gat__</p>
+              <p className="italic">
+                Used to monitor usage of the CPR Website and App. This
+                information allows us to assess how the Website is used.
+                Anonymous data about usage (eg number of visitors, visitors'
+                country, number of times each page is visited) is collected and
```
Comment (joel-wright) [inline frontend/src/pages/cookie-policy.tsx:55]: I'm not sure about this one, but "visitors' country" feels wrong... maybe ignore for now or check with a better grammar checker :joy: 

```suggestion
                Anonymous data about usage (eg number of visitors, visitors'
                countries, number of times each page is visited) is collected and
```

**Review by joel-wright — APPROVED** [review]
A couple of comments, but nothing that can't wait until a follow-up.

File: frontend/src/pages/cookie-policy.tsx (hunk 4 above)
Comment (joel-wright) [reply frontend/src/pages/cookie-policy.tsx:55]: the more I look at this one the less confident I feel about my suggestion :joy:

File: frontend/src/pages/account/index.tsx (hunk 3 above)
Comment (eurolife) [reply frontend/src/pages/account/index.tsx:33]: Will look at this later.

File: frontend/src/pages/cookie-policy.tsx [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+        <LoaderOverlay />
+      ) : (
+        <Layout title={`Climate Policy Radar | ${t('Cookie policy')}`}>
+          <section>
+            <div className="text-content px-4 container mb-12">
+              <h1 className="my-8">Cookie Policy</h1>
+              <p>
+                Read about what cookies are, which cookies Climate Policy Radar
+                (CPR) use on the Climate Policy Radar Website (CPR Website and
+                Ap), and how can you manage your cookies.
```
[the remark this answers is not in the document]
Comment (eurolife) [reply frontend/src/pages/cookie-policy.tsx:22]: good catch. Again I just copy/pasted and didn't actually read it.

File: frontend/src/pages/cookie-policy.tsx (hunk 4 above)
Comment (eurolife) [reply frontend/src/pages/cookie-policy.tsx:55]: This would be something you could raise with Michal, as this is something she wrote.

**Pull request merged**